### ISSUE TITLE
val result 3D AP always 0.00000

### ISSUE THREAD

**Issue by vehxianfish** [wangguojun2018/CenterNet3d#10]
Hi,thank you for your excellent work.
But some error occurs,I hope you can give me some suggestions.
When I train the model.the 3D result is always
```
Car AP@0.70, 0.70, 0.70:
bbox AP:9.8673, 10.1657, 10.5502
bev  AP:0.0000, 0.0005, 0.0005
3d   AP:0.0000, 0.0000, 0.0000
aos  AP:4.86, 5.01, 5.21
Car AP@0.70, 0.50, 0.50:
bbox AP:9.8673, 10.1657, 10.5502
bev  AP:0.0000, 0.0009, 0.0013
3d   AP:0.0000, 0.0005, 0.0006
aos  AP:4.86, 5.01, 5.21
```
Can you help me to points to the bug? thank you~

**Comment by Bovey0809** [comment]
Same, any suggestions?

**Comment by vehxianfish** [comment]
Sorry, Not solve this question yet. Do you?

**Comment by Bovey0809** [comment]
Yes, change the classes in mse loss can solve the problem, the original is only suitable for one class.

**Comment by detectx** [comment]
> Hi,thank you for your excellent work. But some error occurs,I hope you can give me some suggestions. When I train the model.the 3D result is always
> 
> ```
> Car AP@0.70, 0.70, 0.70:
> bbox AP:9.8673, 10.1657, 10.5502
> bev  AP:0.0000, 0.0005, 0.0005
> 3d   AP:0.0000, 0.0000, 0.0000
> aos  AP:4.86, 5.01, 5.21
> Car AP@0.70, 0.50, 0.50:
> bbox AP:9.8673, 10.1657, 10.5502
> bev  AP:0.0000, 0.0009, 0.0013
> 3d   AP:0.0000, 0.0005, 0.0006
> aos  AP:4.86, 5.01, 5.21
> ```
> 
> Can you help me to points to the bug? thank you~

Hello, did you solve this problem?

**Comment by vehxianfish** [comment]
> > Hi,thank you for your excellent work. But some error occurs,I hope you can give me some suggestions. When I train the model.the 3D result is always
> > ```
> > Car AP@0.70, 0.70, 0.70:
> > bbox AP:9.8673, 10.1657, 10.5502
> > bev  AP:0.0000, 0.0005, 0.0005
> > 3d   AP:0.0000, 0.0000, 0.0000
> > aos  AP:4.86, 5.01, 5.21
> > Car AP@0.70, 0.50, 0.50:
> > bbox AP:9.8673, 10.1657, 10.5502
> > bev  AP:0.0000, 0.0009, 0.0013
> > 3d   AP:0.0000, 0.0005, 0.0006
> > aos  AP:4.86, 5.01, 5.21
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Can you help me to points to the bug? thank you~
> 
> Hello, did you solve this problem?

Sorry, Not solve this question yet. Do you?
